api/src/main/java/com/velocitypowered/api/event/EventManager.java

```diff
@@ -57,9 +57,7 @@
    *
    * @param event the event to fire
    */
-  default void fireAndForget(Object event) {
-    fire(event);
-  }
+  void fireAndForget(Object event);
```

Comment:
This is an entirely separate change. (This also technically breaks the API, but seeing as `VelocityEventManager` is the only implementation we care about, that's fine.)

Please break this off into a separate PR.